packages/payload/src/auth/operations/logout.ts

```diff
@@ -36,5 +38,36 @@ export const logoutOperation = async (incomingArgs: Arguments): Promise<boolean>
     }
   }
 
+  if (collectionConfig.auth.disableLocalStrategy !== true && collectionConfig.auth.useSessions) {
+    const userWithSessions = await req.payload.db.findOne<{
+      id: number | string
+      sessions: { id: string }[]
+    }>({
+      collection: collectionConfig.slug,
+      req,
+      where: {
+        id: {
+          equals: user.id,
+        },
+      },
+    })
+
+    if (allSessions) {
+      userWithSessions.sessions = []
+    } else {
+      const sessionsAfterLogout = (userWithSessions?.sessions || []).filter(
+        (s) => s.id !== req.user._sid,
+      )
+
+      userWithSessions.sessions = sessionsAfterLogout
+    }
+
+    await req.payload.db.updateOne({
+      id: user.id,
+      collection: collectionConfig.slug,
+      data: userWithSessions,
```

Comment:
returning: false

Reply:
`returning: false` is a new database option which allows for the update operation to run, but return nothing in response - this is good for performance

Reply:
Oh I see - added!